synadm/cli/user.py

```diff
@@ -500,6 +500,80 @@ def modify(ctx, helper, user_id, password, password_prompt, display_name,
         click.echo("Abort.")
 
 
+# for these placeholders, function similarly to the modify command, but in
+# it's own command.
+# TODO: find 3pid modify command
```

Comment:
We have a 3pid command already, but it's for displaying only: https://synadm.readthedocs.io/en/latest/synadm.cli.user.html#synadm-user-3pid

That I guess is what you are thinking about? Where should we put / and how should we name the modify command for 3pid modify?

Currently 3pid modify is included with the original `user modify` command: https://synadm.readthedocs.io/en/latest/synadm.cli.user.html#cmdoption-synadm-user-modify-t